catalog/pipeline/fbc/0.1/fbc.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: bundle-fbc-pipelinerun
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: fbc
+spec:
+  description: >-
+    Tekton pipeline to release HACBS Snapshot to Quay
+  params:
+    - name: snapshot
+      type: string
+      description: The Snapshot in JSON format
+    - name: enterpriseContractPolicy
+      type: string
+      description: JSON representation of the EnterpriseContractPolicy
+    - name: iibJsonBuildArgs
+      type: string
+      description: IIB JSON Arguments to add a new build
+    - name: iibUrl
+      type: string
+      description: IIB Service URL address
+    - name: iibBuildTimeoutSeconds
+      type: string
+      description: IIB Build timeout seconds
+    - name: iibSecret
```

Comment:
It could use a default.

Reply:
I Think for `timeout` and `url` it makes sense but I don't think the same applies to the secret, as it needs to be manually created beforehand.

Reply:
The name could be fixed though.

Reply:
ok. Added a default "iib-service-secret" name.